osgar/tools/lidarview.py

```diff
@@ -344,6 +344,15 @@ def lidarview(gen, caption_filename, callback=False):
                 if event.key == K_2:
                     frames_step = 20
                     sleep_time = 10
+                if event.key == K_3:
+                    frames_step = 30
+                    sleep_time = 10
+                if event.key == K_4:
+                    frames_step = 40
+                    sleep_time = 10
+                if event.key == K_9:
+                    frames_step = 90
```

Comment:
Is is really useful to have 90x times speed-up? In the old viewer I had 0 and 9 for rewind to the start/end, which is maybe your use case?

Reply:
Well, how do you get into the middle of 4GB log that represents more than 1h? Until we have the scrollbar for random access, this was the best I could come up with.